source/Playnite/ViewModels/SearchViewModel.cs

```diff
@@ -655,7 +655,7 @@ private List<SearchItem> FilterSearchResults(List<SearchItem> toFilter, string f
             return results;
         }
 
-        public static bool MatchTextFilter(string filter, string toMatch)
+        public static bool MatchTextFilter(string filter, string toMatch, bool matchTargetAcronymStart = false)

```

Comment:
matchTargetAcronymStart shouldn't have default value so it's clear how the match is actually done.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3202/commits/46edfe15bb973655782d00d3889b2c5442b83c7e